addons/im_livechat/static/tests/mock_server/mock_models/chatbot_script.js

```diff
@@ -0,0 +1,7 @@
+import { fields, models } from "@web/../tests/web_test_helpers";
+
+export class ChatbotScript extends models.ServerModel {
+    _name = "chatbot.script";
+
+    title = fields.Char({ required: true, default: "Chatbot" });
```

Comment:
Is this really required? I would think the title field would be present as it exists on the server.